crates/nu-test-support/src/lib.rs

```diff
@@ -35,15 +35,15 @@ mod tests {
                 open los_tres_amigos.txt
                 | from-csv
                 | get rusty_luck
-                | str --to-int
+                | str to-int
                 | math sum
                 | echo "$it"
             "#,
         );
 
         assert_eq!(
             actual,
-            r#"open los_tres_amigos.txt | from-csv | get rusty_luck | str --to-int | math sum | echo "$it""#
```

Comment:
No sure how this was working before?

Reply:
We're not actually running this pipeline, just testing that the `pipeline` function produces the expected string 🙂